CMakeModules/AFInstallDirs.cmake

```diff
@@ -21,26 +21,26 @@ endif()
 
 # Data files
 if(NOT DEFINED AF_INSTALL_DATA_DIR)
-  set(AF_INSTALL_DATA_DIR "share/ArrayFire" CACHE PATH "Installation path for data files")
+  set(AF_INSTALL_DATA_DIR "data" CACHE PATH "Installation path for data files")
```

Comment:
Either way, CMake can't find our library automatically(at least on Windows) because we don't stick to standard CMake convention in the installation path because we embed a `v*` folder in the installation path. For example, default installation path on Windows is `C:\Program Files\ArrayFire\v3`. Without that `v3` at the end, our stuff is found automatically without any user intervention.

The reason for changing this is to get a folder heirarchy as close as possible to how previous Windows installation files look like. Apart from that, old location doesn't help CMake in finding out library anyway.